content/industries.yml

```diff
@@ -38,3 +38,12 @@ shopify-supporter: E-commerce
 anova-supporter: IIoT
 auth-keys-supporter: Software
 uswitch-supporter: Consumer Website
+canada-health-infoway-member:
```

Comment:
@oicheryl where can we find the industry for each org?

Reply:
Crunchbase or Linkedin, but quite a few had to be modified as they were inaccurate.